assets/js/common.js

```diff
@@ -35,6 +35,7 @@ function createTable(id, ajaxUrl, orderColumn, orderType, columns) {
       [10, 25, 50, 75, 100, -1],
       [10, 25, 50, 75, 100, 'All'],
     ],
+    pageLength,
```

Comment:
allow overriding this so we can show all a larger page for service areas 
(There were only 12 so 2 hung off the next page)